test/controllers/products_controller_test.rb

```diff
@@ -1,11 +1,25 @@
 require "test_helper"
-
+require 'pry'
 describe ProductsController do
   let(:wizard1) { wizards(:wizard1) }
   let(:wizard_no_products) { wizards(:wizard_no_products) }
   let(:product) { products(:product1) }
-
-  describe "Guest users" do
+  let(:category1) { categories(:category1) } 
+  let(:category5) { categories(:category5) }
+  let(:new_product_params) {
+    {
+      product: {
+        name: "Ancient Amulet",
+        description: "Amulet that also qualifies as an artifact",
+        stock: 1,
+        # photo_url,
```

Comment:
recommend removing commented out lines. Or we can do a cleanup before submitting everything.